common/predictive-text/package.json

```diff
@@ -19,6 +21,20 @@
   "devDependencies": {
     "@types/node": "^10.11.7",
     "chai": "^4.2.0",
+    "karma": "^3.1.1",
+    "karma-browserstack-launcher": "^1.3.0",
+    "karma-chai": "^0.1.0",
+    "karma-chrome-launcher": "^2.2.0",
+    "karma-edge-launcher": "^0.4.2",
+    "karma-firefox-launcher": "^1.1.0",
+    "karma-fixture": "^0.2.6",
+    "karma-html2js-preprocessor": "^1.1.0",
+    "karma-ie-launcher": "^1.0.0",
+    "karma-json-fixtures-preprocessor": "0.0.6",
+    "karma-mocha": "^1.3.0",
+    "karma-mocha-reporter": "^2.2.5",
+    "karma-safari-launcher": "^1.0.0",
+    "karma-teamcity-reporter": "^1.1.0",
```

Comment:
Frankly, I'm really concerned about having two different node environments: one in `common/predictive-text`, and one in `web/`. I think in the future, we may want to look into how these two can share one environment, with the _exact_ same dependencies.